docs/how-to-guides/how-to-integrate-autoware-with-your-vehicle.md

```diff
@@ -0,0 +1,141 @@
+
+!!! note
+At this point, you are now able to run Autoware's Planning Simulator to do a basic test of your vehicle and sensing packages.
+To do so, you need to build and install Autoware using your cloned repository. Follow the [steps for either Docker or source installation](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/)) (starting from the dependency installation step) and then run the following command:
+
+```bash
+ros2 launch autoware_launch planning_simulator.launch.xml vehicle_model:=YOUR_VEHICLE sensor_kit:=YOUR_SENSOR_KIT map_path:=/PATH/TO/YOUR/MAP
+
+```
+
+## 5. Create a `vehicle_interface` package
+
+You need to create an interface package for your vehicle.
+The package is expected to provide the following two functions.
+
+1. Receive command messages from `vehicle_cmd_gate` and drive the vehicle accordingly
+2. Send vehicle status information to Autoware
+
+You can find detailed information about the requirements of the `vehicle_interface` package in the [Vehicle Interface design documentation](https://autowarefoundation.github.io/autoware-documentation/main/design/autoware-interfaces/components/vehicle-interface/).
+You can also refer to TIER IV's [pacmod_interface repository](https://github.com/tier4/pacmod_interface) as an example of a vehicle interface package.
+
+## 6. Launch Autoware
+
+This section briefly explains how to run your vehicle with Autoware.
+
+### Install Autoware
+
+Follow the [installation steps of Autoware](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/).
+
+### Launch Autoware
+
+Launch Autoware with the following command:
+
+```bash
+ros2 launch autoware_launch autoware.launch.xml vehicle_model:=YOUR_VEHICLE sensor_kit:=YOUR_SENSOR_KIT map_path:=/PATH/TO/YOUR/MAP
+```
+
+### Set initial pose
+
+If GNSS is available, Autoware automatically initializes the vehicle's pose.
+
+If not, you need to set the initial pose using the RViz GUI.
+
+1. Click the 2D Pose estimate button in the toolbar, or hit the P key
+2. In the 3D View pane, click and hold the left mouse button, and then drag to set the direction for the initial pose.
+
+### Set goal pose
+
+Set a goal pose for the ego vehicle.
+
+1. Click the 2D Nav Goal button in the toolbar, or hit the G key
+2. In the 3D View pane, click and hold the left mouse button, and then drag to set the direction for the goal pose.
+   If successful, you will see the calculated planning path on RViz.
+
+### Engage
+
+In your terminal, execute the following command.
+
+```bash
+source ~/autoware.YOURS/install/setup.bash
+ros2 topic pub /autoware.YOURS/engage autoware_auto_vehicle_msgs/msg/Engage "engage: true" -1
+```
+
+You can also engage via RViz with "AutowareStatePanel".
+The panel can be found in Panels > Add New Panel > tier4_state_rviz_plugin > AutowareStatePanel.
```

Comment:
For better visibility.
![image](https://user-images.githubusercontent.com/31987104/168573128-c247e51e-5c90-44a1-a3a6-423fb1300a56.png)

```suggestion
The panel can be found in `Panels > Add New Panel > tier4_state_rviz_plugin > AutowareStatePanel`.
```